index.js

```diff
@@ -40,8 +43,14 @@ function matchLicense(licenseString) {
     }
     // For single-line license, check if it's a known license id.
     if (matchingLicenses.length === 0 && !/[\n\f\r]/.test(licenseString) && licenseString.length < 100) {
-        var licenseName = licenseString.trim()
-        matchingLicenses.push(licenseIndex[licenseName] || {name: licenseName, id: null})
+        var licenseName = normalizeText(licenseString)
+        // If there's an extra "license" on the end of the name, drop it ("MIT License" -> "MIT").
+        var license = licenseIndex[licenseName] || licenseIndex[licenseName.replace(/ licen[sc]e$/, "")]
```

Comment:
Considered that, but think this is more robust since either way works. I didn't want to review every normalized name in SPDX.